Kitodo/src/main/java/org/kitodo/production/forms/createprocess/SimpleAdditionalDetailsTableRow.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.forms.createprocess;
+
+import java.io.Serializable;
+import java.lang.reflect.Method;
+
+import org.kitodo.api.dataeditor.rulesetmanagement.MetadataViewInterface;
+import org.kitodo.api.dataeditor.rulesetmanagement.SimpleMetadataViewInterface;
+import org.kitodo.api.dataformat.IncludedStructuralElement;
+import org.kitodo.exceptions.NoSuchMetadataFieldException;
+
+abstract class SimpleAdditionalDetailsTableRow extends AdditionalDetailsTableRow implements Serializable {
```

Comment:
why is there a "simpleAdditionalDetails..." when the "additionalDetails" is nowhere used?
Merge those two classes